src/recipes/tooling-editor-plugin.md

```diff
@@ -0,0 +1,56 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Editor Plugin
+
+Using `EditorPlugin` types is very similar to the process used when
+[writing plugins in GDScript](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/making_plugins.html). However,
+like Custom Classes, they
+are registered automatically by godot-rust and cannot be enabled/disabled in the Project Settings plugins pane.
+
+Plugins written in GDScript are automatically disabled if they have a code error, but because Rust is a compiled language,
+you cannot introduce compile-time errors because the library binary will simply not be emitted on the compiler finding an error.
+This is not the case for run-time errors.
+
+
+## Creating an EditorPlugin
+
+```rs
+#[derive(GodotClass)]
+#[class(tool, editor_plugin, base=EditorPlugin)]
+struct MyEditorPlugin {
+    #[base]
+    base: Base<EditorPlugin>,
+}
+
+impl IEditorPlugin for MyEditorPlugin {
+    fn init(base: Base<EditorPlugin>) -> Self {
+        // this is a prime candidate to be replaced by "init"
+        Self { base }
+    }
```

Comment:
I don't understand this comment -- if it's a prime candidate to be replaced, why is that function even here?